### PR TITLE
Test commit

### PR DESCRIPTION
**Pull request by LuisitoFCantagojr**
- blabla

### CODE REVIEW

**Comment by LuisitoFCantagojr** [comment]
Please check outline-not-fix

File: src/app/add-post/add-post.component.html [hunk 1]
```diff
@@ -21,7 +21,13 @@
             <span class="text-center text-sm-start" id="image-error"></span>
         </div>
         <div class="mt-3 mt-sm-0 d-flex justify-content-center justify-content-sm-end align-items-center">
-            <app-button type='button' class="btn-secondary header rounded" text="Submit" (btnClick)="post()"></app-button>
+
+
+        <div class="custom-padding">
+            <app-button type='button' class="btn btn-outline-primary" text="Submit" (btnClick)="post()"></app-button>
```
Comment (nikkipanda-dev) [inline src/app/add-post/add-post.component.html:27]: Remove "btn" in class="btn btn-outline-primary". Should only be btn-outline-primary. Button component already has btn class

Comment (LuisitoFCantagojr) [reply src/app/add-post/add-post.component.html:27]: I see hold on

File: src/app/post/post.component.html [hunk 2]
```diff
@@ -4,16 +4,16 @@
...
         <div class="clearfix text-start">
             <img src="{{ post.parsed_image }}" class="post-image {{ post.parsed_image ? 'd-block' : 'd-none' }} col-md-6 float-md-start mb-3 me-md-3"/>
             <p>{{ post.body ? post.body : '' }}</p>
         </div>
     </div>
-    <div class="card-footer d-flex flex-wrap flex-row justify-content-center justify-content-sm-end align-items-center">
+    <div  class="card-footer d-flex flex-wrap flex-row justify-content-center justify-content-sm-end align-items-center">
         <small class="text-muted me-0 me-sm-2">{{ post.parsed_date ? post.parsed_date : '' }}</small>
-        <button text="Delete" class="btn btn-dark btn-sm header" data-bs-toggle="modal" data-bs-target="#static-modal" data-target-type="delete" id="{{ post.id + '-card' }}" (click)="showAlert()">Delete</button>
+        <button text="Delete" class="btn btn btn-outline-secondary" data-bs-toggle="modal" data-bs-target="#static-modal" data-target-type="delete" id="{{ post.id + '-card' }}" (click)="showAlert()">Delete</button>
```
Comment (nikkipanda-dev) [inline src/app/post/post.component.html:17]: Remove extra btn. Should be btn btn-outline-secondary only

File: src/app/add-post/add-post.component.html (hunk 1 above)
Comment (LuisitoFCantagojr) [reply src/app/add-post/add-post.component.html:27]: Hi please see new commit

File: src/app/post/post.component.html (hunk 2 above)
Comment (LuisitoFCantagojr) [reply src/app/post/post.component.html:17]: Hi please see new commit with only one btn

**Pull request merged**